packages/atomic/src/components/atomic-date-facet/atomic-date-facet.tsx

```diff
@@ -92,7 +92,22 @@ export class AtomicDateFacet implements AtomicComponentInterface {
     this.controller.sortBy(criterion);
   }
 
+  // TODO: improve loading style
+  public renderLoading() {
+    return (
+      <div class="loading">
```

Comment:
It would be much better to create an `atomic-loading` component and reuse it instead of duplicating the CSS and DOM for the loading.